kodi_addon_checker/check_addon.py

```diff
@@ -78,6 +78,10 @@ def start(addon_path, config=None):
 
             _check_artwork(addon_report, addon_path, addon_xml, file_index)
 
+            if config.is_enabled("check_complex_files"):
+                max_line_count = config.configs.get("max_line_count", 80)
```

Comment:
@enen92 here i have taken a default value for `max_line_count`so if it is not present in any config file then it will use a default value.
Should I leave it to 80 ?

Reply:
I'd keep it much lower than that as a default like 10. However there is an issue. 
You're blindingly counting the lines of code and you basically neglect the presence of comments. Some addon devs keep and header at the top with a copyright message that will increase the number of lines in the file but won't get executed by the interpreter

Reply:
See:
https://github.com/enen92/screensaver.atv4/blob/master/entrypointscript.py

The tool would report 26 lines of code when in fact the entrypoint file is only composed of 3

Reply:
ok so i'll make another function for checking the line which will ignore docstring and comments